test/units/TEST-07-PID1.private-pids.sh

```diff
@@ -147,13 +147,13 @@ testcase_unpriv() {
     rm -rf /tmp/TEST-07-PID1-private-pids-proc
 
     # Now verify the behavior with masking - units should fail as PrivatePIDs=yes has no graceful fallback.
-    if [[ "$HAS_EXISTING_SCSI_MOUNT" == "no" ]]; then
+    if [[ "$HAS_EXISTING_SCSI_MOUNT" == "no" ]] && [ -d /proc/scsi ]; then
```

Comment:
This wouldn't work. We want to mask something under /proc/, so that the unprivileged PrivatePIDs= invocation fails

Reply:
This whole test makes no sense